apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -761,12 +729,15 @@ suite('conversation.js >', function() {
 
     setup(function() {
       banner = document.getElementById('messages-subject-max-length-notice');
-      Compose.showSubject();
+
+      Compose.isSubjectVisible = true;
```

Comment:
Will add, but that will be ```false``` anyway, so it won't change anything for this test :)